ci-operator/step-registry/cucushift/installer/rehearse/aws/cases/all-regions/provision/cucushift-installer-rehearse-aws-cases-all-regions-provision-commands.sh

```diff
@@ -0,0 +1,534 @@
+# init result file
+# -------------------------------------------------------------------------------------
+echo "Creating result file ..."
+t=$(mktemp)
+while IFS= read -r region; do
+    cat > ${t} << EOF
+{
+  "region": "${region}",
+  "cluster_name": "$(get_cluster_name "${region}")",
+  "install_dir": "$(get_install_dir "${region}")",
+  "infra_id": "NA",
+  "install_result": "NA",
+  "health_check_result": "NA",
+  "destroy_result": "NA",
+  "metadata": ""
+}
+EOF
+    cat <<< "$(jq  --argjson info "$(<${t})" --arg region $region '. += {($region): $info}' "${RESULT}")" > ${RESULT}
+done < ${REGIONS_LIST}
+
+# -------------------------------------------------------------------------------------
+# Create cluster
+# -------------------------------------------------------------------------------------
+total=$(cat $REGIONS_LIST | wc -l)
+i=0
+while IFS= read -r region; do
+    set +o errexit
+    let i+=1
+    # init result
+    cluster_name=$(get_cluster_name "${region}")
+    install_dir=$(get_install_dir "${region}")
+    mkdir -p ${install_dir}
+
+    echo "================================================================"
+    echo "Creating cluster [${region}][${cluster_name}], ${i}/${total}"
+    echo "================================================================"
+    create_install_config $region $cluster_name $install_dir
+
+    # create ignition configs
+    openshift-install create ignition-configs --dir ${install_dir} &
+    wait "$!"
+    install_ret="$?"
+    ret=$((ret+install_ret))
+    if [ $install_ret -ne 0 ]; then
+      echo "Failed ignition configs, saving metadata.json ... "
+      report_install_result "${region}" "FAIL"
+      continue
+    else
+      echo "Created ignition configs, saving metadata.json and infraid ... "
+
+      cp ${install_dir}/metadata.json ${SHARED_DIR}/metadata.${region}.json
+
+      metadata_b64="$(cat ${install_dir}/metadata.json | base64 -w0)"
+      infra_id="$(cat ${install_dir}/metadata.json | jq -r '.infraID')"
+      cat <<< "$(jq --arg metadata ${metadata_b64} --arg region ${region} '.[$region].metadata = $metadata' "${RESULT}")" > ${RESULT}
+      cat <<< "$(jq --arg i ${infra_id} --arg region ${region} '.[$region].infra_id = $i' "${RESULT}")" > ${RESULT}
+    fi
+
+    # create cluster
+    openshift-install create cluster --dir ${install_dir} 2>&1 | grep --line-buffered -v 'password\|X-Auth-Token\|UserData:' &
+    wait "$!"
+    install_ret="$?"
+    ret=$((ret+install_ret))
+
+    if [ $install_ret -ne 0 ]; then
+      report_install_result "${region}" "FAIL"
+      continue
+    else
+      report_install_result "${region}" "PASS"
+    fi
+
+    echo "--- Health check ---"
+
+    if [[ -f ${install_dir}/auth/kubeconfig ]]; then
+      export KUBECONFIG=${install_dir}/auth/kubeconfig
+      health_check
+      health_ret=$?
+      ret=$((ret+health_ret))
+
+      if [ $health_ret -ne 0 ]; then
+        report_health_check_result "${region}" "FAIL"
```

Comment:
Missing `continue` here ?

Reply:
health check is the last step, it will go to the next iteration automatically